### ISSUE TITLE
Extraneous space after family name in citations indices

### ISSUE THREAD

**Issue by geoffreyjones** [plk/biblatex#376]
The new biblatex 3.4 index mechanism seems to add an extraneous space after author family name in citations indices, at least as tested in conjunction with the `imakeidx` package.  E.g., example `22-indexing-subentry.tex` creates the following `name-title.idx` file (noting extraneous spaces after Knuth and Aristotle):

```
\indexentry{Knuth , Donald E. !Computers & Typesetting@\emph  {Computers \& Typesetting}}{1}
\indexentry{Knuth , Donald E. !The TeXbook@\emph  {\TeX book, The}}{1}
\indexentry{Knuth , Donald E. !METAFONTbook, The@\emph  {METAFONTbook, The}}{1}
\indexentry{Knuth , Donald E. !METAFONT: The Program@\emph  {METAFONT: The Program}}{1}
\indexentry{Aristotle ,  !De Anima@\emph  {De Anima}}{1}
\indexentry{Aristotle ,  !Poetics@\emph  {Poetics}}{1}
\indexentry{Aristotle ,  !Physics@\emph  {Physics}}{1}
\indexentry{Aristotle ,  !Rhetoric of {Aristotle}, The@\emph  {Rhetoric of {Aristotle}, The}}{1}
```

**Comment by plk** [comment]
I have found the issue and am looking at it. Tricky expansion code which is different between index and normal name formats.

**Comment by plk** [comment]
Thank you for reporting this - I believe that this is fixed in the 3.4 release which is in the development folder on Sourceforge. Please could you try it and confirm? If using biber, you will need the 2.5 development version, also on Sourceforge.

**Comment by geoffreyjones** [comment]
Regressed 3.4/2.5 release on 1000 citations, 300 page document. All confirmed good. Many thanks.
